client/components/social-buttons/google.js

```diff
@@ -87,25 +87,28 @@ class GoogleLoginButton extends Component {
 			.then( gapi =>
 				gapi.client
 					.init( {
-						client_id: this.props.clientId,
-						scope: this.props.scope,
 						fetch_basic_profile: this.props.fetchBasicProfile,
 						ux_mode: this.props.uxMode,
 						redirect_uri: this.props.redirectUri,
 					} )
-					.then( () => {
-						this.setState( { isDisabled: false } );
-
-						const googleAuth = gapi.auth2.getAuthInstance();
-						const currentUser = googleAuth.currentUser.get();
-
-						// handle social authentication response from a redirect-based oauth2 flow
-						if ( currentUser && this.props.uxMode === 'redirect' ) {
-							this.props.responseHandler( currentUser, false );
-						}
-
-						return gapi; // don't try to return googleAuth here, it's a thenable but not a valid promise
-					} )
+					.then( () =>
+						gapi.auth2.init( {
+							client_id: this.props.clientId,
+							scope: this.props.scope,
+						} ).then( () => {
+							this.setState( { isDisabled: false } );
+
+							const googleAuth = gapi.auth2.getAuthInstance();
+							const currentUser = googleAuth.currentUser.get();
+
+							// handle social authentication response from a redirect-based oauth2 flow
+							if ( currentUser && this.props.uxMode === 'redirect' ) {
+								this.props.responseHandler( currentUser, false );
+							}
+
+							return gapi; // don't try to return googleAuth here, it's a thenable but not a valid promise
+						} )
+					)
```

Comment:
The whole approach for fixing that issue was suggested [here](https://github.com/google/google-api-javascript-client/issues/316#issuecomment-313742954).